get-started.md

```diff
@@ -78,3 +61,39 @@ Check out the [projects page](reference/projects.md) for more information about
 3. **Broadcast updates in multiple places**. Different people have different preferred methods of engagement and communication on 2i2c projects. For this reason, don't assume that posting an update or question in a single location will reach everybody that you wish to reach. Do not hesitate to post questions or ask for feedback in multiple places (e.g., GitHub, Slack, or even email). However, try to keep information in a single place to have a single source of truth.
 
 4. **Update often**. Err on the side of updating the team more often than not.  Nobody should be surprised when something gets done - especially nothing major. Your work should be as transparent to others as possible!
+
+
+## Where information is located
+
+### General organizational information / documents / brainstorms
+
+We also have some organizational documents in a shared Google folder called "2i2c". Your 2i2c address will give you access to this folder (or if you do not have access, please request it!):
+
+[https://drive.google.com/drive/folders/1ABxxSFycGfCzQc9czfwer_dat-GVi4jw?usp=sharing](https://drive.google.com/drive/folders/1ABxxSFycGfCzQc9czfwer_dat-GVi4jw?usp=sharing)
+
+The folder is organized into a few top-level folders that *should* be relatively self-explanatory, but don't hesitate to ask for a quick guide.
+
+### Todos, project plans and management
+
+2i2c uses [the Projects page](../projects/index.md) to keep track of the major projects that it is working on. That page has links to the location of project-specific to-do lists, deliverables, etc.
+
+More organization-specific discussion and items we track can also be found in [the private 2i2c `meta` repository](https://github.com/2i2c-org/meta).
+
+Generally speaking the daily specifics of activity on a project are recorded *outside* of the team compass, as they are more dynamic. The important thing is that the team compass make it clear *where* to find this information, and that the information is kept up-to-date for all.
+
+
+## Marketing and communication
+
+There are three main ways that 2i2c communicates with the outside world:
+
+- Twitter: [@2i2c_org](https://twitter.com/2i2c_org)
+  - Currently, there is nobody activately monitoring the Twitter account. We use it to post links to blog posts or major announcements in the newsletter.
+- Newsletter: [using Mailchimp](https://mailchimp.com/)
```

Comment:
How frequently do we send newsletters and what do we usually say there? Do we have records of previous newsletters?

Reply:
mostly it is just mirroring our blog posts - a way to get the information to a more dedicated group of folks.